lib/lotus/helpers/html_helper/self_closing_html_node.rb

```diff
@@ -0,0 +1,25 @@
+module Lotus
+  module Helpers
+    module HtmlHelper
+      class SelfClosingHtmlNode
+        def initialize(name, attributes)
+          @name       = name
+          @attributes = attributes
+        end
+
+        def to_s
+          %(<#{ @name }#{attributes}>)
```

Comment:
Okay, fair enough. Since the generator generates `application.html` with tag `<!doctype HTML>`. I am happy with it.